CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+13.4.0
+------
+* Skip CSRF protection if a valid signed JWT token is present as we trust Shopify to be the only source that can sign it securely.
```

Comment:
Not critical, but I think it's helpful to reference, or link to, the PR.

Reply:
Thanks Andy! Just added the reference to the PR